data/categories.txt

```diff
@@ -346,7 +346,7 @@ cs:2Bar|pivo|pití|jídlo|voda|pitná voda|tekutiny
 sk:2Bar|pivo|pitie|jedlo|voda|pitná voda|tekutiny
 nl:2Bar|2pub|2café|bier|drinken|eten
 zh-Hant:1酒吧|酒館|飲酒|PUB|吧台|飲食
-pl:2Bar|2pub|piwo|napoje|jedzenie
+pl:2Bar|alkohole|drinki|napoje|jedzenie
```

Comment:
You've removed "pub" here, so poles won't find bars if they enter "pub" in the search field. Is that right?